lib/diagnostic.js

```diff
@@ -38,19 +43,28 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = ['name', 'species', 'breed', 'vocalization'];
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
-const Pet = function () {
-  /* Your answer here */
+const useVocalization = function (){
+  return this._vocalization;
+};
+//answer to #5
+
+const Pet = function (name, species, breed, vocalization) {
+  this.name = name;
+  this.species = species;
+  this. breed = breed;
+  this._vocalization= vocalization;
+  this.useVocalization = useVocalization;
```

Comment:
This works, but in the future please don't rename named identifiers so that the tests work correctly.  e.g., this diagnostic asks for a method called ```vocalize``` not ```useVocalization```.